README.md

```diff
@@ -97,3 +97,39 @@ generation. In that case we excluded the file from being generated and manually
 
 The hope going forward is that we can continue to use the generator with minimal manual
 intervention.
+
+### WASM - WebAssembly
+
+Requirements:
+
+- `wasm32-unknown-unknown` rust target.
+- `wasm-bindgen-cli` installed.
+- `binaryen` installed for `wasm-opt` and `wasm2js`.
+
+```bash
+rustup target add wasm32-unknown-unknown
+cargo install -f wasm-bindgen-cli
+brew install binaryen
+```
+
+#### Build
+
+```bash
+# dev
+cargo build -p bitwarden-wasm -p bitwarden --target wasm32-unknown-unknown --features wasm-bindgen
+wasm-bindgen --target bundler --out-dir languages/js/wasm ./target/wasm32-unknown-unknown/debug/bitwarden_wasm.wasm
+wasm-bindgen --target nodejs --out-dir languages/js/wasm/node ./target/wasm32-unknown-unknown/debug/bitwarden_wasm.wasm
+
+# release
+cargo build -p bitwarden -p bitwarden-wasm --target wasm32-unknown-unknown --features wasm-bindgen --release
+wasm-bindgen --target bundler --out-dir languages/js/wasm ./target/wasm32-unknown-unknown/release/bitwarden_wasm.wasm
+wasm-bindgen --target nodejs --out-dir languages/js/wasm/node ./target/wasm32-unknown-unknown/release/bitwarden_wasm.wasm
```

Comment:
Are we going to support using the wasm builds in node while also offering the node-api support?

Reply:
I would like to offer it at least while we explore if we can transition off the napi bindings internally.